src/runner/htmlRunner.ts

```diff
@@ -0,0 +1,20 @@
+import { IOptions, Result } from '..'
+import { Context } from '../types'
+
+export const ERROR_HANDLING_SCRIPT = `<script>
+  window.onerror = (msg, url, lineNum) => {
+    window.parent.postMessage("Line " + Math.max(lineNum - 5, 0) + ": " + msg, "*");
+  };
+</script>\n`
```

Comment:
Is there a reason why this particular newline is represented by `\n` while the previous newlines are represented by literal line breaks?

Reply:
Mainly for readability; the last `\n` is to signal that the user's code begins on the next line.